### ISSUE TITLE
Option to configure content negotiation

### ISSUE THREAD

**Issue by iannesbitt** [DataONEorg/mnlite#58]
By default, mnlite has `application/ld+json` as the first priority in the list of content types to accept from the server. Sometimes, however, content negotiation will redirect to different locations with different JSON-LD formatting embedded or served by each. Thus, the `Accept:` header content should be configurable per node. Default should be the following, as this is what is in use currently:
```
application/ld+json, application/json;q=0.5, text/html;q=0.8, application/xhtml+xml;q=0.8
```

**Comment by iannesbitt** [comment]
Correction—this is the `Accept:` header in use now:

```
application/ld+json,text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8
```

**Comment by iannesbitt** [comment]
I forgot that `DEFAULT_REQUEST_HEADERS` is actually set in `settings.py`, meaning that this is actually possible in the current version. In order to override this global setting, a node just needs the following local configuration option in its `settings.json` file in the project folder:

```json
{
    ...
    "DEFAULT_REQUEST_HEADERS": {"Accept": "text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8"}
}
```

Closing this issue as already implemented.

**Comment by iannesbitt** [comment]
This was implemented in #38.
